lib/logstash/outputs/loggly.rb

```diff
@@ -59,6 +59,12 @@ class LogStash::Outputs::Loggly < LogStash::Outputs::Base
   # https://www.loggly.com/docs/source-groups/
   config :tag, :validate => :string, :default => "logstash"
 
+  # Retry count
+  config :retry_count, :validate => :number, :default => 5
+
+  # Can Retry
+  config :can_retry, :validate => :boolean, :default => false
```

Comment:
Could we simplify these new settings into a single setting, such as `send_attempts` or something, with a default of 1 and having only values >=1 as valid. Users wishing multiple attempts could set this number to 5 or something.

(just an idea)